yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -56,8 +59,8 @@ fi
 if [ "$#" -lt 4 ]; then
     echo "Usage: $0 <pg_connection_string> <schema_list> <assessment_metadata_dir> <pgss_enabled> [iops_capture_interval]"
     exit 1
-elif [ "$#" -gt 5 ]; then
-    echo "Usage: $0 <pg_connection_string> <schema_list> <assessment_metadata_dir> <pgss_enabled> [iops_capture_interval]"
+elif [ "$#" -gt 6 ]; then
```

Comment:
this would be needed even on line 60, I guess, right?

Reply:
in general, if we're printing out out the "usage" and exiting, it's better to include the complete usage help.